### ISSUE TITLE
Plugin not working?

### ISSUE THREAD

**Issue by ChestRockwel** [cobalamin/obsidian-remarkable#5]
I just got this installed in Obsidian, but when I use the hotkey - ctrl` - it doesn't do anything.  reSnap works on its own, and the directory pointer seems right - /home/prc/reSnap/reSnap.sh

Is there something I'm missing here?

**Comment by cobalamin** [comment]
Does it work through the command palette? Maybe there's an issue with the shortcut. If it also doesn't work, let me know.

**Comment by ChestRockwel** [comment]
Issue fixed - I think that for some reason I had the old release - loaded up the new one and it seems to be working now.

**Comment by ChestRockwel** [comment]
### Resolved this issue - I had created the reSnap folder as root - changing permissions fixed it - left this here for anyone else who has this problem though ###

So, I thought things were working, but it seems I was wrong. Whenever I try to run the plugin, I get "Could not insert your rM drawing! Is your tablet connected and reachable at the configured address?

reSnap works just fine - I should note that I'm using WSL to run this, but everything else in the chain of stuff - reSnap/passwordlessSSH - seems fine. 

Obsidian debug giving this:


node:internal/errors:464 Uncaught (in promise) Error: spawn /home/prc/reSnap/reSnap.sh EACCES
    at __node_internal_captureLargerStackTrace (node:internal/errors:464:5)
    at __node_internal_errnoException (node:internal/errors:594:12)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)


### New Issue ###

Basically when I run the script - and it's working!!! - the image doesn't actually appear in Obsidian until I reload the app? Is there any way to fix this? The .png appears in my "Remarkable Snaps" folder in my Vault immediately after running the script, but for some reason the image doesn't point to the correct .png which should be there.

Beneath the link I get a "rM drawing blah blah blah.png is not created yet. Click to create." prompt that just creates a blank page in the root of my vault, but I can see - in my file manager - the png is there. Any suggestions as to what's going on?

I am so close to getting this awesome plugin to work.

**Comment by cobalamin** [comment]
Sorry for the late response. Has anything changed about your newer issue by now? Do you have the current version of Obsidian running? It doesn't sound like a problem I've ever encountered before.

**Comment by cobalamin** [comment]
@Folcon I just saw this small code change you made in your fork, may this fix you made have anything to do with @ChestRockwel's new issue? https://github.com/cobalamin/obsidian-remarkable/commit/2c96ce1b6a9095018aab12b81637fc0cee1af5f4
